.ki-scripts/README.md

```diff
@@ -0,0 +1,45 @@
+# Kitty Items Scripts
+
```

Comment:
Can we add a short description of what this document is all about? It's not obvious when jumping into it without context.